src/.eslintrc.json

```diff
@@ -2,5 +2,8 @@
 	"extends": ["plugin:@typescript-eslint/recommended-requiring-type-checking", "../eslint.base.json"],
 	"parserOptions": {
 		"project": ["./tsconfig.json"]
+	},
+	"rules": {
+		"@typescript-eslint/no-unused-vars": ["error", { "ignoreRestSiblings": true }]
```

Comment:
With our open source projects, I try not to deviate too much (if any) from the default recommended rules. Otherwise, we could go down a long trail of enabling/disabling rules to create a custom set for our company’s unique preferences.

How hard would it be to not change this rule’s default behavior and write code that adheres to the default recommendations.

Reply:
It would just be more verbose, I'd need to just hand `validateOptions` a new object with just the options specified rather than pulling them out of the object altogether like I have currently.

Reply:
I think that’s worth doing in order to preserve the default ESLint recommended rules.